test/python/quantum_info/states/test_statevector.py

```diff
@@ -1164,6 +1164,19 @@ def test_statevector_draw_latex_regression(self):
         self.assertTrue(latex_string.startswith(" |0\\rangle"))
         self.assertNotIn("|1\\rangle", latex_string)
 
+    def test_statevctor_iter(self):
+        """Test iteration over a state vector"""
+        empty_vector = []
+        dummy_vector = [1, 2, 3]
+        empty_sv = Statevector([])
+        sv = Statevector(dummy_vector)
+
+        # Assert that successive iterations behave as expected, i.e., the
+        # iterator is reset upon each exhaustion of the corresponding
+        # collection of elements.
+        for _ in range(2):
+            self.assertEqual(empty_vector, [x for x in empty_sv])
+            self.assertEqual(dummy_vector, [x for x in sv])
 
 if __name__ == "__main__":
```

Comment:
The style check probably wants an extra blank line between the end of the class and the test runner statement.  In general, you can just run `tox -e black` to run the formatter though, provided you've got `tox` in your Python environment.

Reply:
Thanks for the useful information provided! I did run the test suite with `tox` but wasn't aware how to run the formatting checks. The PR will be updated accordingly.